libraries/azurerm_virtual_machine.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# TODO: we will have a name collision with the existing implementation
```

Comment:
Could we reword this to be something we can do? Some action item?

Reply:
I'll remove. We resolved it by switching to `azurerm` instead of `azure`. We should probably have a comment on why the name is different or standardize on `azurerm`.